DigitalLearningSolutions.Web/Scripts/searchSortFilterAndPaginate/sort.ts

```diff
@@ -10,13 +10,15 @@ export function setUpSort(onSortUpdated: VoidFunction): void {
 export function sortSearchableElements(
   searchableElements: ISearchableElement[],
 ): ISearchableElement[] {
-  const sortBy = getSortBy();
+  const sortByValue = getSortBy();
+  const sortByIteratees = document.querySelector('[data-sort-by-multiple]') ? sortByValue.split(',') : [sortByValue];

```

Comment:
This name's a bit opaque... what about fieldsToSortBy?

Reply:
Ah, I see the lodash docs use `iteratees`, but I've never heard it before. Is it a commonly used name?

Reply:
Ha yeah, I'm sure in some circles it might be used.
This was simply to align to the lodash conventions but not required.
I've gone with your suggestion.